### PR TITLE
Submission service tests

### PR DESCRIPTION
**Pull request by SebastianWild**
## Checklist

* [ ] This PR does not change text that hasn't been signed off with the RKI. Have a look at the pinned issue [440](https://github.com/corona-warn-app/cwa-app-ios/issues)
* [x] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
* [x] If this PR comes from a fork, please [Allow edits from maintainers](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
* [ ] Set a speaking title. Format: {task_name} (closes #{issue_number}). For example: Use logger (closes # 41)
* [ ] [Link your Pull Request to an issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) - Pull Requests that are not linked to an issue with you as an assignee will be closed.
* [ ] Create Work In Progress [WIP] pull requests only if you need clarification or an explicit review before you can continue your work item.
* [ ] Make sure that your PR is not introducing _unnecessary_ reformatting (e.g., introduced by on-save hooks in your IDE)

## Description

This PR adds some small unit tests + refactoring on how we handle the processing of `ENTemporaryExposureKey`s before the submission process.
Basically, some logic is extracted from `ExposureSubmissionService` and unit tested separately.

### CODE REVIEW

**Review by inf2381 — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Services/Risk/Calculation/RiskCalculation.swift [hunk 1]
```diff
@@ -137,18 +137,19 @@ enum RiskCalculation {
 		summary: CodableExposureDetectionSummary,
 		configuration: SAP_ApplicationConfiguration
 	) -> Double {
-
+		// "Fig" comments below point to figures in the docs: https://github.com/corona-warn-app/cwa-documentation/blob/master/solution_architecture.md#risk-score-calculation
```
Comment (johannesrohwer) [inline src/xcode/ENA/ENA/Source/Services/Risk/Calculation/RiskCalculation.swift:140]: very helpful 👍

**Review by johannesrohwer — APPROVED** [review]
thanks for adding tests!
